settings.py

```diff
@@ -24,5 +26,14 @@
 USE_TZ = False
```

Comment:
The `USE_TZ` setting is set to `False`. Consider enabling timezone support (`USE_TZ = True`) to ensure that datetime fields are stored in UTC and converted to the local timezone in the application.